Quick/DSL/World+DSL.swift

```diff
@@ -97,6 +97,6 @@ extension World {
     }
 
     internal func pending(description: String, closure: () -> ()) {
-        print("Pending: \(description)")
+        print("Pending: \(description)", appendNewline: false)
```

Comment:
We need to handle pending examples in a nicer way, but for the time being, shouldn't this include a new line? Each pending example should be on its own line in the test output.